Scripts/Tracking/Query/ObscuranceQuery.cs

```diff
@@ -0,0 +1,91 @@
+﻿namespace VRTK.Core.Tracking.Query
+{
+    using System;
```

Comment:
One other thing i forgot to mention. (nitpicky too)

I usually structure it so it goes

Unity
System
VRTK

So put the `using System...` stuff under the `using Unity...` stuff